internal/pkg/levee/levee.go

```diff
@@ -40,10 +51,25 @@ var Analyzer = &analysis.Analyzer{
 		fieldtags.Analyzer,
 		source.Analyzer,
 		suppression.Analyzer,
+		buildssa.Analyzer,
```

Comment:
We aren't using the `earpointer.Analyzer` here, which means we won't get the benefits of the `analysis` package ordering analysis passes.  That's okay for now, but for future considerations where the EAR analysis may need to rely on object `Fact`s to operate between packages, we will want the ordering imposed and encapsulated by that `Analyzer`, and we don't want to need knowledge of those internal workings in this package.

It may be preferable instead to have the EAR analyzer be a Required analyzer here, rather than calling across with `p := earpointer.Analyze(ssainput)`.  That is, we could handle the `useEAR = false` case with that analyzer acting as a no-op, returning a trivial or sentinel result, which indicates that the `runPropagation` approach should be used as a fall-back.  Conversely, `useEAR = true`, we would unpack results returned by the EAR analyzer, i.e. `pass.ResultOf[earpointer.Analyzer].(*earpointer.WhateverResultType)` and emit findings based on that report.

(For consistency, we would want as future work to refactor the propagation approach to a separate analyzer and have the `levee.Analyzer` become a shim that only handles delegation and report processing, but that would be well out of scope for any PR integrating the `earpointer` and `levee` packages.)

Reply:
Now EAR is an analysis pass. If it is not enabled, then the pass doesn't nothing and simply return an empty state. The flag controlling this has moved to the "config.go".